tests/FakeItEasy.Analyzer.Tests/Helpers/DiagnosticVerifier.cs

```diff
@@ -114,9 +81,9 @@ private static void VerifyDiagnosticResults(IEnumerable<Diagnostic> actualResult
                     {
                         var message =
                             FormattableString.Invariant(
-                                $@"Expected:
-A project diagnostic with No location
-Actual:
+                                $@"Expected:
+A project diagnostic with No location
+Actual:

```

Comment:
Could it have something to do with the `core.autocrlf` setting in git? Mine is set to `true`.